services/server/events.ts

```diff
@@ -43,7 +43,11 @@ export const getEvents = async (odataFilter: string): Promise<EventInfo[]> => {
     );
     return [];
   }
-
+  process.env.MICROSOFT_OAUTH_CLIENT_ID,
+    console.log(
+      "🚀 ~ file: events.ts:47 ~ getEvents ~ process.env.MICROSOFT_OAUTH_CLIENT_ID,:",
```

Comment:
I would add an explanation here

"Missing token"

Reply:
That was an accidental commit, reverted it in new commits.